crates/sui-indexer/src/models/addresses.rs

```diff
@@ -14,6 +14,7 @@ use diesel::result::Error;
 #[derive(Queryable, Debug)]
 #[diesel(primary_key(account_address))]
 pub struct Addresses {
+    pub id: i64,
```

Comment:
nit: should this be u64?

Reply:
this is a PG thing that they do not support unsigned integer, which is odd indeed.
https://stackoverflow.com/questions/20810134/why-unsigned-integer-is-not-available-in-postgresql